spec/routing/affiliate_events_routing_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe AffiliateEventsController, type: :routing do
+  describe "routing" do
+
+    it "routes to #index" do
+      expect(:get => "/affiliate_events").to route_to("affiliate_events#index")
```

Comment:
Unfortunately scaffold doesn't generate specs according to the newest rails style. So we have to manually clean them to avoid offenses.
Use: expect(get: '/affiliate_events').to route_to("affiliate_events#index")